### PR TITLE
Replace PrefixPool with IPPool in excludedPrefixesServer

### LINKED ISSUE #389: Rework prefixpool to work with using RoaringBitmap
## Motivation

We've migrated prefix pool from mono repo to sdk https://github.com/networkservicemesh/sdk/tree/master/pkg/tools/prefixpool, but It looks like many things can be refactored and simplified with using this library:
https://github.com/networkservicemesh/sdk/blob/master/pkg/networkservice/ipam/point2pointipam/server.go#L31

### PR DESCRIPTION
**Pull request by artem-belov**
Fixes #389
Benchmark results #768

### CODE REVIEW

File: pkg/networkservice/common/excludedprefixes/server_test.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ func TestNewExcludedPrefixesService(t *testing.T) {
 }
 
 func TestCheckReloadedPrefixes(t *testing.T) {
-	prefixes := []string{"172.16.1.0/24", "10.32.0.0/12", "10.96.0.0/12"}
+	prefixes := []string{"10.32.0.0/12", "10.96.0.0/12", "172.16.1.0/24"}
```
Comment (denis-tingaikin) [inline pkg/networkservice/common/excludedprefixes/server_test.go:62]: Why tests have changed? Looks like it is more correct, but could you please clarify this :)

File: pkg/tools/ippool/ippool.go [hunk 2]
```diff
@@ -553,6 +582,102 @@ func (node *treeNode) maximumNode() *treeNode {
...
+		// exclude the last /64 network if it is not full
+		if end.low != math.MaxUint64 {
+			end.high--
+		}
+
+		// get available prefixes bigger than /64
+		for start.high <= end.high {
+			z := trailingZeros(start.high)
+
+			if z == 64 {
```
Comment (denis-tingaikin) [inline pkg/tools/ippool/ippool.go:617]: Can we make `64` as a constant?

File: pkg/tools/ippool/ippool.go [hunk 3]
```diff
@@ -230,6 +235,30 @@ func (tree *IPPool) PullP2PAddrs(exclude ...*IPPool) (srcNet, dstNet *net.IPNet,
...
+	}
+	for it.node.Left != nil {
+		it.node = it.node.Left
+	}
+
+	for node := it.Next(); node != nil; node = it.Next() {
+		prefixes = append(prefixes, node.getPrefixes(clone.ipLength)...)
+	}
+
+	return
```
Comment (denis-tingaikin) [inline pkg/tools/ippool/ippool.go:259]: I wonder why linter does not trigger at this line. 
Can we do not use return without values?

**Review by denis-tingaikin — APPROVED** [review]

File: pkg/networkservice/common/excludedprefixes/server_test.go (hunk 1 above)
Comment (artem-belov) [reply pkg/networkservice/common/excludedprefixes/server_test.go:62]: IPPool sort subnets with the bit order within it's RB tree, so I changed prefixes order to check in the test

File: pkg/tools/ippool/ippool.go (hunk 2 above)
Comment (artem-belov) [reply pkg/tools/ippool/ippool.go]: Done

File: pkg/tools/ippool/ippool.go (hunk 3 above)
Comment (artem-belov) [reply pkg/tools/ippool/ippool.go]: Done

**Pull request merged**